Makefile

```diff
@@ -2,28 +2,53 @@
 ## Install gcn2scimma
 ##
 INSTALL_DIR   := /usr/local/gcn2hop
-PYTHON_FILES  := gcn2hop Utils.py 
+WORKING_DIR	  := ./gcn2hop
+PYTHON_FILES  := gcn2hop,Utils.py
 INSTALL_FILES := $(patsubst %,$(INSTALL_DIR)/%,$(PYTHON_FILES))
-
-.PHONY: all install install_dir test
+PYTHON = python
+
+.PHONY: all install test
+
+.PHONY: help
+help :
+	@echo
+	@echo 'Commands:'
+	@echo
+	@echo '  make test                  run unit tests'
+	@echo '  make lint                  run linter'
+	@echo '  make format                run code formatter, giving a diff for recommended changes'
+	@echo '  make doc                   make documentation'
+	@echo '  make install				install module
```

Comment:
Please fix the indentation stuff for the help messages here.

Reply:
I will change it.